frontend/src/concepts/pipelines/content/tables/experiment/ExperimentTableRow.tsx

```diff
@@ -24,11 +24,20 @@ const ExperimentTableRow: React.FC<ExperimentTableRowProps> = ({
 }) => {
   const { namespace } = usePipelinesAPI();
 
+  const isArchived = window.location.href.includes('/archived');
+
   return (
     <Tr>
       <CheckboxTd id={experiment.experiment_id} isChecked={isChecked} onToggle={onToggleCheck} />
       <Td dataLabel="Experiment">
-        <Link to={experimentRunsRoute(namespace, experiment.experiment_id)} state={{ experiment }}>
+        <Link
+          to={
+            isArchived
+              ? experimentArchivedRunsRoute(namespace, experiment.experiment_id)
```

Comment:
`experimentArchivedRunsRoute` is the same as `experimentRunsRoute` except with a querystring of runType, so why not just dynamically add the querystring to either an additional parameter or just to a template string here instead of making a new function?

Reply:
Great point, thank you - updated